keyboards/citrus/erdnuss65/erdnuss65.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2023 Citrus Lab
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
```

Comment:
This file should be removed.

Reply:
> 应删除此文件。

Really? But after deleting it, I can't seem to compile properly

Reply:
> 您是否也进行了上述建议的更改？

Yes, but I couldn't compile after removing erdnuss65.h

Reply:
Try `qmk clean` as well. And just to confirm, `erdnuss65.h` file is not being included anywhere?
eg. ```
#include "erdnuss65.h"
```

Reply:
> 也试试。只是为了确认，文件没有包含在任何地方？例如。`qmk clean``erdnuss65.h`
> 
> ```
> #include "erdnuss65.h"
> ```

After using `qmk clean`, I can compile normally, thank you